aldryn_reversion/utils.py

```diff
@@ -27,7 +27,7 @@ def build_obj_repr(obj):
     Returns a unicode string of object Model name and its text representation.
     """
     return "{0}: '{1}'".format(force_text(obj._meta.model.__name__),
-                                force_text(obj))
+                               force_text(obj))
```

Comment:
I notice that you're only loading force_text(), what about older Django's who only have force_unicode()?